.travis.yml

```diff
@@ -21,7 +20,7 @@ before_install:
   - echo 'opcache.enable_cli=1' >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
   - travis_retry composer self-update
 
-install: travis_retry composer update --optimize-autoloader --prefer-dist --prefer-stable --no-progress --no-interaction -vv $COMPOSER_FLAGS
+install: travis_retry composer install --optimize-autoloader --prefer-dist --no-progress --no-interaction -vv $COMPOSER_FLAGS
```

Comment:
I would prefer to leave the update here, as it prevents from failing builds because of incompatilities between the versions defined in the composer.lock and the versions supporting the php versions the tests running on.